code/modules/detectivework/tools/rag.dm

```diff
@@ -126,7 +126,8 @@
 						SPAN_DANGER("You smother \the [target] with \the [src]!")
 					)
 					//it's inhaled, so... maybe CHEM_INJECT doesn't make a whole lot of sense but it's the best we can do for now
-					var/trans_amt = reagents.trans_to_mob(target, amount_per_transfer_from_this, CHEM_INJECT)
+					// ^ HA HA HA
```

Comment:
Minor nit, but personally would prefer those comments get deleted rather than leaving chains of banter in the source.

Reply:
the "ha ha ha" is directly from aurora's equivalent for the code, iirc, but yeah I can remove it